### PR TITLE
Update media-dynamic-update.md

### PR DESCRIPTION
**Pull request by SteveDiAcetis**

### CODE REVIEW

File: windows/deployment/update/media-dynamic-update.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ Properly updating the installation media involves a large number of actions oper
 
 This table shows the correct sequence for applying the various tasks to the files. For example, the full sequence starts with adding the servicing stack update to WinRE (1) and concludes with adding the Dynamic Update for Setup to the new media (26).
 
+**Note**: Starting in February 2021, the latest cumulative update (LCU) and servicing stack update (SSU) will be combined and distributed in the Microsoft Update Catalog as a new, combined cumulative update. For steps 1, 9 and 18 below that require the SSU for updating the installation media, the combined cumulative update should be used.
```
Comment (illfated) [inline windows/deployment/update/media-dynamic-update.md:67]: Your existing note format is fine in coding samples and scripts, but MS Docs codestyle requires notes to be added as colored Note blobs in the standard document text:
```suggestion
> [!NOTE]
> Starting in February 2021, the latest cumulative update (LCU) and servicing stack update (SSU) will be combined and distributed in the Microsoft Update Catalog as a new, combined cumulative update. For steps 1, 9 and 18 below that require the SSU for updating the installation media, the combined cumulative update should be used.
```
Thank you for adding the useful information.

**Comment by jaimeo** [comment]
Note to copy editors: do not use "LCU" or "SSU." Spell out (and do not capitalize) those terms. Also we do not use "above" and "below" as directional terms within a document. Always refer to a specific section or table, etc. instead.

**Comment by illfated** [comment]
Thank you for the guidance note. Quite useful to know and remember.

**Comment by illfated** [comment]
I am pondering the distinction between the current phrase "Starting in February 2021, [ … ]" and the shorter form 'Starting February 2021,' — and whether it is better to use the current form than the alternative. Maybe the shortened form is simply popularized by marketing announcements, so it may not apply to the writing style of documents like this one?

**Comment by jaimeo** [comment]
"Starting in..." is clearer. And better for Loc.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by SteveDiAcetis** [comment]
@jaimeo I'd like to merge these changes. There will be future updates once the link for 2001 2C is available. Further, once more backporting happens, we'll update the servicing stack page.

**Pull request merged**